Documentation/SETTINGS.md

```diff
@@ -1,21 +1,31 @@
 Create a settings.ini file, paste this inside:
 
 [settings]
+PRODUCTION=True/False
+
 OAUTH_PROVIDER=sanic_oauth.providers.GoogleClient
-OAUTH_REDIRECT_URI=<your_domain_uri>/oauth
+OAUTH_REDIRECT_URI=/oauth
 OAUTH_SCOPE=email
 OAUTH_CLIENT_ID=<your_client_id>
 OAUTH_CLIENT_SECRET=<your_client_secret>
 
 WTF_CSRF_SECRET_KEY=<generated_key>
 
-DOMAIN_NAME=<your_domain>
+DOMAIN_NAME=<your_production_domain>
+LOCAL_HOST=http://localhost:8000
 
 WHITELISTED_EMAIL_1=<email_domain_1>
 WHITELISTED_EMAIL_2=<email_domain_2>
 ...
 WHITELISTED_EMAIL_N=<email_domain_n>
 
+MYSQL_HOST=<mysql_host_name>
+MYSQL_DB=<mysql_db_name>
+MYSQL_USER=<mysql_user_name>
+MYSQL_PASSWORD=<mysql_password>
+MYSQL_ROOT_PASSWORD=<mysql_root_password>
```

Comment:
Is this needed anywhere?

Reply:
For the K8 container to connect to the mysql K8 container. In development, they'll connect through docker-compose/

Reply:
Yes, actually I meant this particular one. The `MYSQL_ROOT_PASSWORD` one. Is this used anywhere? The rest is OK.

Reply:
It should be a mandatory field for db initialisation via docker-compose.

Reply:
Yes. Right. As an environment variable. But does it make sense here in `settings.ini`?

Reply:
Not for us, but perhaps someone else might deploy the app via Docker, and they would've to hide the values in .env/.ini file.
(Removed)